components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -123,17 +123,19 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
     }, [logsEmitter, props.workspaceId, workspaceInstance?.id, workspaceInstance?.status.phase]);
 
     return (
-        <div className="rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-800 flex flex-col">
+        <div className="rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-800 flex flex-col mb-8">
```

Comment:
Ended up adding the margin here - tested all three views. Also I think moving forward we should make the logs be centered in the screen. It's too off down there at the bottom.